src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -76,6 +77,28 @@ public final void testDefaultFlawless() throws IOException {
     }
 
     @Test
+    public final void testSourcePathsOneFile() throws IOException {
+        final CheckstyleAntTask antTask = getCheckstyleAntTask();
+        final FileSet examinationFileSet = new FileSet();
+        examinationFileSet.setFile(new File(getPath(FLAWLESS_INPUT)));
+        final Path sourcePath = new Path(antTask.getProject());
+        sourcePath.addFileset(examinationFileSet);
+        antTask.addPath(sourcePath);
+        antTask.execute();
```

Comment:
@MEZk @rnveach @romani I've added a custom `RootModuleChecker` to the tests and check that is was called with appropriate files. Is it acceptable, or there is better way?